File: 11. Pendalaman String/C. Caesar Cipher/c_caesar_cipher.go
package main

import "fmt"

func main() {
    var S string
    var K int
    fmt.Scan(&S, &K)
    for i := 0; i < len(S); i++ {
        fmt.Print(string((int(S[i]) + K - int('a')) % 26 + int('a')))
    }
    fmt.Println();
}

File: 11. Pendalaman String/C. Caesar Cipher/c_caesar_cipher.java
import java.util.Scanner;

public class c_caesar_cipher {
    public static void main(String args[]) {
        Scanner scan = new Scanner(System.in);
        String S = scan.next();
        int K = scan.nextInt();
        scan.close();
        for (int i = 0; i < S.length(); i++) {
            char s = S.charAt(i);
            System.out.print((char) ((s - 'a' + K) % 26 + 'a'));
        }

    }
}
